snmp/manifest.json

```
@@ -29,7 +29,7 @@
   "assets": {
     "monitors": {},
     "dashboards": {
-      "Device Dashboard": "assets/dashboards/device_dashboard.json",
+      "Interface Performance": "assets/dashboards/interface_performance.json",
```

```suggestion
      "Device Dashboard": "assets/dashboards/interface_performance.json",
```
Let's keep the shortname the same, it does not affect the dashsboard content but for db purposes

---

I'd like to change the name used in routing, and it seems that keeping name as they were keeps also names in routing as they were, meaning they are still reachable at `app.datadoghq.com/https://dd.datad0g.com/screen/integration/Interface%20Performance`

---

If it's complicated to do that, I can check with product and see if it's ok to see a different name in the navigation bar

---

Checked with product, ok to keep old navigation name

---

So I'm a bit confused, this dashboard is leading to 404 https://dd.datad0g.com/dashboard/lists/preset/3?q=Interface

---

Sorry for the confusion, last days before holidays we decided to change the dashboard name and clean it up, as this is a new dashboard and we haven't communicated on it yet, but we will soon.

The new link should work by now, can you retry? All DB should be synced now. What was happening is that there was a delay on resolution of the address, depending on the pod that was asked for it, it was returning the new address or 404 - 404 being returned 1/4 times.